src/System.IO.MemoryMappedFiles/tests/MemoryMappedFile.CreateFromFile.Tests.cs

```diff
@@ -603,37 +603,6 @@ public void FileInUse_CreateFromFile_FailsWithExistingReadWriteMap()
         }
 
         /// <summary>
-        /// Test exceptional behavior when trying to create a map for a non-shared file that's currently in use.
-        /// </summary>
-        [Fact]
-        public void FileInUse_CreateFromFile_FailsWithExistingNoShareFile()
-        {
-            // Already opened with a FileStream
-            using (TempFile file = new TempFile(GetTestFilePath(), 4096))
-            using (FileStream fs = File.Open(file.Path, FileMode.Open, FileAccess.ReadWrite, FileShare.None))
-            {
-                Assert.Throws<IOException>(() => MemoryMappedFile.CreateFromFile(file.Path));
-            }
-        }
-
-        /// <summary>
-        /// Test to validate we can create multiple concurrent read-only maps from the same file path.
-        /// </summary>
-        [Fact]
-        public void FileInUse_CreateFromFile_SucceedsWithReadOnly()
-        {
-            const int Capacity = 4096;
-            using (TempFile file = new TempFile(GetTestFilePath(), Capacity))
-            using (MemoryMappedFile mmf1 = MemoryMappedFile.CreateFromFile(file.Path, FileMode.Open, null, Capacity, MemoryMappedFileAccess.Read))
-            using (MemoryMappedFile mmf2 = MemoryMappedFile.CreateFromFile(file.Path, FileMode.Open, null, Capacity, MemoryMappedFileAccess.Read))
-            using (MemoryMappedViewAccessor acc1 = mmf1.CreateViewAccessor(0, Capacity, MemoryMappedFileAccess.Read))
-            using (MemoryMappedViewAccessor acc2 = mmf2.CreateViewAccessor(0, Capacity, MemoryMappedFileAccess.Read))
-            {
-                Assert.Equal(acc1.Capacity, acc2.Capacity);
-            }
-        }
```

Comment:
The first removed test was combined with the `FileInUse_CreateFromFile_FailsWithExistingReadWriteFile` test so that the FileShare is passed as INlineData.

The second test asserts the concurrent access which no longer passes with FileShare.None.